include/deal.II/base/mpi_consensus_algorithms.h

```diff
@@ -440,6 +445,34 @@ namespace Utilities
         clean_up_and_end_communication();
       };
 
+      /**
+       * A serial fall back for the above classes to allow programming
+       * independently if MPI is used or not.
```

Comment:
Even though I am not a native English speaker, I think this should be 
```suggestion
       * independently of whether MPI is used or not.
```

Reply:
Done!